ui/src/literature/components/DOILinkAction.jsx

```diff
@@ -1,30 +1,62 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { List } from 'immutable';
+import { Button, Menu } from 'antd';
 
 import ListItemAction from '../../common/components/ListItemAction';
-import EventTracker from '../../common/components/EventTracker';
+import DropdownMenu from '../../common/components/DropdownMenu';
+import IconText from '../../common/components/IconText';
+import DOILink from './DOILink';
+import DOIMaterial from './DOIMaterial';
 
 class DOILinkAction extends Component {
+  static renderIconTextForDOI() {
+    return <IconText text="DOI" type="link" />;
+  }
+
+  renderDoiDropdownItem() {
+    const { dois } = this.props;
+    return dois.map(doi => (
+      <Menu.Item key={doi.get('value')}>
+        <DOILink doi={doi.get('value')}>
+          {doi.get('value')}
+          <DOIMaterial material={doi.get('material')} />
+        </DOILink>
+      </Menu.Item>
+    ));
+  }
+
+  renderDoiDropdown() {
```

Comment:
consistent naming would be better, so everwhere upper case `DOI`, except variable names.